did_document_format.proto

```diff
@@ -15,7 +15,7 @@ message VerificationMethod {
   string id = 1;
   VerificationType type = 2;
   string controller = 3;
-  string publicKeyMultibase = 4;
+  string public_key_multibase = 4;
```

Comment:
Why are you changing the field to snake-case format?
This is going to affect the final schema output of the did document some of the other languages.

Please revert to previous camelCase format.

Reply:
Please see screenshots in slack. When using camelCase, verificationmethods and serviceendpoint keywords are generated without camelCase. Using snake_case in proto is [recommended](https://protobuf.dev/programming-guides/style/#message-field-names).